Parsers/Sysmon-v9-Parser.txt

```diff
@@ -1,27 +1,26 @@
 // KQL Sysmon Event Parser
-// Last Updated Date: Nov 25, 2019
-// Sysmon Version : 10.2.0.0
+// Last Updated Date: Dec 9, 2019
+// Sysmon Binary Version : 9.10 , Schema Version: 4.21
```

Comment:
Lets leave in the Sysmon Version, same as I suggested above.  Helps make sure users understand what version this supports.

Reply:
Added details and renamed files